aquadoggo/src/lib.rs

```diff
@@ -59,3 +60,11 @@ fn init() {
         let _ = env_logger::builder().is_test(true).try_init();
     }
 }
+
+fn info_or_print(message: &str) {
+    if log_enabled!(Level::Info) || log_enabled!(Level::Debug) || log_enabled!(Level::Trace) {
+        info!("{message}");
+    } else {
+        println!("{message}");
```

Comment:
Double string interpolation probably not required

Reply:
Both macros expect a string literal like: `"{}"`